controllers/runner_controller.go

```diff
@@ -775,6 +775,10 @@ func newRunnerPod(template corev1.Pod, runnerSpec v1alpha1.RunnerConfig, default
 			Name:  "RUNNER_GROUP",
 			Value: runnerSpec.Group,
 		},
+		{
+			Name:  "DOCKER_ENABLED",
```

Comment:
You are not passing `DISABLE_WAIT_FOR_DOCKER` to the runner ? I don't think this PR works ?

Reply:
PR was poorly named